app/api/resources/user.py

```diff
@@ -93,13 +93,13 @@ def get(cls, user_id):
         """
         # Validate arguments
         if not OtherUser.validate_param(user_id):
-            return messages.USER_ID_IS_NOT_VALID, HTTPStatus.BAD_REQUEST
+            return messages.USER_ID_IS_NOT_VALID, HTTPStatus.FORBIDDEN
```

Comment:
FORBIDDEN are not raising if we are using  `users/<int:user_id>`. On routing `localhost:5000/users/abc` then this error pop up `The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.`
So either we should remove `users/<int:user_id>` to `users/<user_id>` or remove `FORBIDDEN status`.
@isabelcosta

Reply:
Very interesting 🤔 I am not sure what the "best" route is here?!
I wonder if this check is even needed... What would you suggest from the 2 options you suggested @satya7289 ?

Also... @ramitsawhney27 @m-murad

Reply:
I would suggest remove the `FORBIDDEN` status. @isabelcosta